ts/src/profiler.ts

```diff
@@ -80,12 +80,7 @@ function isDeployment(deployment: any): deployment is Deployment {
   return (deployment.projectId === undefined ||
```

Comment:
Coverage says this is not covered by tests?

Reply:
Adjusted tests and fixed this.